src/Marten/Schema/DuplicatedField.cs

```diff
@@ -38,9 +37,19 @@ public DuplicatedField(EnumStorage enumStorage, MemberInfo[] memberPath) : base(
                 PgType = "varchar";
             }
             else if (MemberType.IsDateTime())
+            {
```

Comment:
Are all the Datetime/DatetimeOffset query tests passing?

Reply:
Yes.